feat: update react-native-spokestack to 5@beta (#1)
spokestack/react-native-spokestack-tray: README.md

  // ...
```

</details>

## Android installation

<details>
  <summary>Android details</summary>

### ASR Support

> The example usage uses the system-provided ASRs (`AndroidSpeechRecognizer` and `AppleSpeechRecognizer`). However, `AndroidSpeechRecognizer` is not available on 100% of devices. If such a device is supported, use Spokestack ASR instead by setting the `profile` to a Spokestack profile using the `profile` prop.

Reviewer: ```suggestion
The example usage uses the system-provided ASRs (`AndroidSpeechRecognizer` and `AppleSpeechRecognizer`). However, `AndroidSpeechRecognizer` is not available on 100% of devices. If your app supports a device that doesn't have built-in speech recognition, use Spokestack ASR instead by setting the `profile` to a Spokestack profile using the `profile` prop.
```

            }
        }
      }}
      // The NLU models are downloaded and then cached
      // when the app is first installed.

>       // These files can made manually or

Reviewer: ```suggestion
      // These files can be made manually or
```
...though good luck with that :)

• `Optional` **minHeight**: number (Default: **170**)

_Defined in [src/SpokestackTray.tsx:171](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/SpokestackTray.tsx#L171)_

Minimum height for the tray

### nlu

• **nlu**: SpokestackConfig[\"nlu\"]

_Defined in [src/SpokestackTray.tsx:128](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/SpokestackTray.tsx#L128)_

> The NLU Tensorflow-Lite model (.tflite), JSON metadata, and NLU vocabulary (.txt)

Reviewer: ```suggestion
The NLU Tensorflow Lite model (.tflite), JSON metadata, and NLU vocabulary (.txt)
```

> All 3 fields accepts 2 types of values.

Reviewer: ```suggestion
All 3 fields accept 2 types of values.
```

Starting height for tray

### style

• `Optional` **style**: Animated.WithAnimatedValue<StyleProp<ViewStyle\>\>

_Defined in [src/SpokestackTray.tsx:237](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/SpokestackTray.tsx#L237)_

This style prop is passed to the tray's container

### ttsFormat

• `Optional` **ttsFormat**: TTSFormat (Default: **TTSFormat.TEXT**)

_Defined in [src/SpokestackTray.tsx:239](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/SpokestackTray.tsx#L239)_

The format for the text passed to Spokestack.synthesize

### voice

• `Optional` **voice**: string (Default: **"demo-male"**)

_Defined in [src/SpokestackTray.tsx:241](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/SpokestackTray.tsx#L241)_

A key for a voice in Spokestack ASR, passed to Spokestack.synthesize

### wakeword

• `Optional` **wakeword**: SpokestackConfig[\"wakeword\"]

_Defined in [src/SpokestackTray.tsx:275](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/SpokestackTray.tsx#L275)_

> The NLU Tensorflow-Lite models (.tflite) for wakeword.

Reviewer: ```suggestion
The NLU Tensorflow Lite models (.tflite) for wakeword.
```

> All 3 fields accepts 2 types of values.

Reviewer: ```suggestion
All 3 fields accept 2 types of values.
```

```

---

### isInitialized

• `Const` **isInitialized**: isInitialized = Spokestack.isInitialized

_Defined in [src/index.ts:24](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/index.ts#L24)_

Returns whether Spokestack has been initialized

```js
console.log(`isInitialized: ${await isInitialized()}`)
```

---

### isStarted

• `Const` **isStarted**: isStarted = Spokestack.isStarted

_Defined in [src/index.ts:32](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/index.ts#L32)_

Returns whether the speech pipeline has been started

```js
console.log(`isStarted: ${await isStarted()}`)
```

---

### addEventListener

• `Const` **addEventListener**: addListener = Spokestack.addEventListener

_Defined in [src/index.ts:48](https://github.com/spokestack/react-native-spokestack-tray/blob/fd8dea5/src/index.ts#L48)_

Bind to any event emitted by the native libraries
The events are: "recognize", "partial_recognize", "error", "activate", "deactivate", and "timeout".
See the bottom of the README.md for descriptions of the events.

```js
useEffect(() => {
  const listener = addEventListener('recognize', onRecognize)

>   // Unsubsribe by calling remove when components are unmounted

Reviewer: ```suggestion
  // Unsubscribe by calling remove when components are unmounted
```
